.ci/static-checks.sh

```diff
@@ -310,6 +325,8 @@ static_check_go_arch_specific()
 
 		info "Forcing ${linter} version ${linter_version}"
 		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin "${linter_version}"
+		command -v $linter &>/dev/null || \
+			die "$linter command not found. Ensure that \"\$GOPATH/bin\" is in your \$PATH."
```

Comment:
Nit: Some usages of `die`  are using single quotes to quote the error message, which will not need backslashes to escape the dollar char. So should all use single quotes?

Reply:
Hi @liubin , this line is the only exception because the tool's name is in a variable (`$linter`). The other error messages I had the tool's name hard-coded in the text so I could use single-quotes. Or am I missing something? Thanks for reviewing it!